from0k2bp.tex

```diff
@@ -589,24 +599,24 @@
 a lot of other high quality resources too.
 
 As a preparation for using these ideas in practice, here is how the
-proof works for a (slightly) simpler case, that of Schnorr's identity
-protocol (which is basically the same as the Schnorr signature, except
-the interactive form, and ignoring messages). To review, the basic
+proof works for a (slightly) simpler case, namely for Schnorr's identity
+protocol\footnote{It is basically the same as the Schnorr signature, except
+the interactive form and ignoring messages}. To review, the basic
 structure is:
 
 Prover starts with a public key $P$ and a corresponding private key $x$ s.t. $P=xG$.
 
 Prover wishes to prove in zero knowledge, that he knows $x$.
 
-P → V: $R$ (a new random curve point, but P knows $k$ s.t. $R=kG$)
+$\mathcal{P}$ → $\mathcal{V}$: $R$ (a new random curve point, but $\mathcal{P}$ knows $k$ s.t. $R=kG$)
 
-V → P: $e$ (a random scalar)
+$\mathcal{V}$ → $\mathcal{P}$: $e$ (a random scalar)
 
-P → V: $s$ (which P calculated from the equation $s=k+ex$)
+$\mathcal{P}$ → $\mathcal{V}$: $s$ (which $\mathcal{P}$ calculated from the equation $s=k+ex$)
 
-Note: the \textbf{transcript} referred to above, would here be: $(R, e, s)$.
+Note: the \textbf{transcript} referred to above, would be here: $(R, e, s)$.
```

Comment:
No, '... here be' is strictly better than '... be here' .. here :)

Reply:
Reverted.